doc/userguide/rules/dns-keywords.rst

```diff
@@ -22,6 +22,17 @@ used in a rule all contents following it are affected by it.  Example:
 The dns_query keyword affects all following contents, until pkt_data
 is used or it reaches the end of the rule.
 
+dns_response
+------------
+
+The **dns_response** keyword allows you to match IPs in A record responses.
+
+ Example:
+ alert dns any any -> any any (msg:"Test dns_reputation option";
```

Comment:
Lets not make any reference to 'reputation' here as it's a simple match. Lets reserve reputation for when we can make large number of ip's.